packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2810,6 +2822,13 @@ class InputDecoration {
   ///  * [suffixIcon], which is the same but on the trailing edge.
   final Widget? prefixIcon;
 
+  /// The alignment for [prefixIcon].
+  ///
+  /// Used to align the [prefixIcon] realtive to the position of the text box.
```

Comment:
This should be **relative**, am I correct?

Reply:
Yep thanks a lot